lang/lang-es.js

```diff
@@ -0,0 +1,560 @@
+    supportDesktop: 'La clase virtual no es compatible con dispositivos móviles. Es aconsejable usar un pc o portátil.',
+    supportDesktopOnly: 'Lo sentimos, el modo de presentación solo es compatible con pcs o portátiles. Los dispositivos móviles no son totalmente compatibles.',
+    teacherSafariBrowserIssue: 'El navegador Safari no admite la funcionalidad de presentación. Por favor, use Chrome o Firefox.',
+    techsupport: 'Apoyo técnico',
+    testingbrowser: 'Test de compatibilidad del navegador',
+    testinginternetspeed: 'Test de velocidad de conexión',
+    testingmichrophone: 'Test de micrófono',
+    testingspeaker: 'Test de altavoces',
+    testingwebcam: 'Test de webcam',
+    Text: 'Texto',
+    Text_Limit_Exeed: 'Advertencia: Demasiado texto',
+    TextEditor: 'Editor de texto',
+    textPlaceholder: 'Introducir texto aquí',
+    Time: 'Tiempo',
+    Tmyclose: 'El docente puede cerrar esta encuesta en cualquier momento.',
+    total: 'Total',
+    totalprogress: 'Progreso total',
+    tpAudioTest: 'Test de audio',
+    transferControls: 'Controles de transferencia',
+    Triangle: 'Triángulo',
+    Udocument: 'Subir documento',
+    Unauthenticated: 'Error: Acceso no válido',
+    unmuteAll: 'Dejar de silenciar todo',
+    uploadsession: 'Por favor espere a que finalice todo el proceso',
+    uploadvideo: 'Subir vídeo',
+    uploadedsession: 'Su sesión ha finalizado. Puede cerrar la ventana. '
+    + ' o cierre esta ventana emergente para comenzar una nueva sesión.',
+    usermissing: 'Falta el alumno',
+    userList: "Lista de usuarios",
+    upvote: 'Voto a favor',
+    instructorVideo: 'Video del docente',
+    validateurlmsg: 'URL no válida',
+    VCE2: 'No tiene permiso para grabar esta sesión',
+    VCE4: 'Faltan datos de registro',
+    VCE5: 'Incapaz de registrar datos.',
+    VCE6: 'Falta el módulo del curso.',
+    VideodbHeading: 'Panel del vídeo',
+    videoInput: 'Entrada de Vídeo',
+    videooff: 'Vídeo desactivado',
+    videoon: 'Vídeo activado',
+    videoquality: 'Calidad de video',
+    virtualclassnoteHeader: 'Añadir nota',
+    VotedSoFar: 'Votos recibidos hasta ahora',
+    votesuccess: 'Su voto ha sido enviado con éxito. El resultado se mostrará en breve.',
+    votesuccessPbt: 'Voto enviado con éxito, no se le permite ver el resultado.',
+    Vwllbshrshortly: 'El video se compartirá en breve',
+    viewall: 'Ver todo',
+    waitmsgconnect: 'Por favor, espere un momento. La aplicación está intentando conectarse.',
+    watstdrespo: 'Esperando la respuesta del alumno',
+    wbrtcMsgChrome: 'Puede hacer clic en el botón denegar para no compartir su micrófono y cámara con Congrea.' + 'o haga clic en el botón Permitir para compartirlo.',
+    wbrtcMsgFireFox: 'Puedes hacer clic en  "Compartir dispositivos seleccionados"' + ' para compartir tu micrófono y cámara con otros usuarios.',
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [This line has a length of 146. Maximum allowed is 120.](https://app.codacy.com/gh/congrea/virtualclass/pullRequest?prid=5529949)